README.md

```diff
@@ -3,6 +3,112 @@ python-krux-boto
 
 Krux Python class built on top of [Krux Stdlib](https://staticfiles.krxd.net/foss/docs/pypi/krux-stdlib/) for interacting with [Boto](http://boto.readthedocs.org/en/latest/) and [Boto3](http://boto3.readthedocs.org/en/latest/index.html)
 
+CRITICAL NOTE about versions!
+-----------------------------
+
+Version 0.0.7 went through a major refactoring work and thus is not compatible with applications written with `krux-boto` version 0.0.6. The mechanism to fetch AWS credential is changed and may cause your application to use a different credentials than intended. Therefore, version 0.0.7 **should not be used**. Please use either 0.0.6 or 1.0.0 version.
+
+### Updating from 0.0.6 to 1.0.0
+
+In version 0.0.6, `krux_boto.Boto` object took an `argparse.ArgumentParser` object as an optional parameter for the constructor. This approach has been abandoned. `krux_boto.Boto` object now expects 4 parameters listed below. Therefore, following change is required to get your application working with version 1.0.0.
```

Comment:
Let's move the code below the application quick start, but leave the big warning above in place with a link to the code below - This only affects people who are doing maintenance on existing apps and the 80-90% scenario is new code.